lib/docx/body-reader.js

```diff
@@ -356,23 +356,48 @@ function BodyReader(options) {
 
 
     function readTable(element) {
-        var propertiesResult = readTableProperties(element.firstOrEmpty("w:tblPr"));
-        return readXmlElements(element.children)
-            .flatMap(calculateRowSpans)
-            .flatMap(function(children) {
-                return propertiesResult.map(function(properties) {
-                    return documents.Table(children, properties);
-                });
-            });
+        var tablePropertiesReadResult = readTableProperties(element.firstOrEmpty("w:tblPr"));
```

Comment:
Again here, these patches are quite invasive. Is there an option to package them in an extension?

Reply:
I can try but it was really complicated to get this function working correctly. The whole `ReadResult` thing they're using is really convoluted